### PR TITLE
Show data list posts

### PR DESCRIPTION
**Pull request by mvn-tamnguyen4-dn**
## Description
A few sentences describing the overall goals of the pull request's commits.
## Reference Requirements
- Product Backlog ID: PBI-9
- Task ID: 9.2

## Migrations
NO

## Impacted Areas in Application
lang/en/post.php

## SQL query and `explain`
select `posts`.`id`, `users`.`name`, `type`, `posts`.`content`, `posts`.`created_at`, (select count(*) from `comments` where `posts`.`id` = `comments`.`post_id` and `comments`.`deleted_at` is null) as `comments_count` from `posts` left join `users` on `posts`.`user_id` = `users`.`id` where `posts`.`deleted_at` is null group by `posts`.`id`
image

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -20,4 +20,5 @@
     Route::resource('users', 'UserController', ['except' => ['create', 'store']]);
     Route::get('/', 'HomeController@index')->name('home.index');
     Route::resource('books', 'BookController');
+    Route::resource('posts', 'PostController');
```
Comment (at-xinhnguyen) [inline routes/web.php]: User có thể xem được post không em

File: resources/views/backend/posts/index.blade.php [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+                <td>
+                @switch($post->type)
+                  @case(1)
+                    Review
+                    @break
+                  @case(2)
+                    Status
+                    @break
+                  @case(3)
+                    Find book
```
Comment (at-xinhnguyen) [inline resources/views/backend/posts/index.blade.php]: trans 3 text này em nhé

File: resources/views/backend/posts/index.blade.php [hunk 3]
```diff
@@ -32,48 +32,39 @@
...
-                <td>Orion Daugherty</td>
-                <td>Review</td>
-                <td>Incidunt voluptatem officiis esse quis in...</td>
-                <td>2018-01-16 09:16:58</td>
-                <td>10</td>
+                <td>{{ $post->id }}</td>
+                <td>{{ $post->name }}</td>
+                <td>
+                @switch($post->type)
+                  @case(1)
```
Comment (at-xinhnguyen) [inline resources/views/backend/posts/index.blade.php]: em nên tạo 3 const trong model POST 
const REVIEW_TYPE = 1;
...
...

File: app/Http/Controllers/Admin/PostController.php [hunk 4]
```diff
@@ -14,6 +15,18 @@ class PostController extends Controller
...
+            'posts.id',
+            'users.name',
+            'type',
+            'posts.content',
+            'posts.created_at',
+        ];
+        $posts = Post::leftJoin('users', 'posts.user_id', '=', 'users.id')
+                    ->select($fields)
+                    ->withCount('comments')
+                    ->groupBy('posts.id')
```
Comment (at-xinhnguyen) [inline app/Http/Controllers/Admin/PostController.php]: ở đây k cần group by đâu em

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**